pom.xml

```diff
@@ -161,6 +162,18 @@
             <version>${junit5.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Is this dependency actually required? I thought all the tests were junit5?

Reply:
Ah yes good point (was part of experimenting to see why the tests weren't being ran) - I've removed the `vintage-engine` dependency - cheers